src/main/java/mate/academy/hibernate/relations/service/impl/ActorServiceImpl.java

```diff
@@ -1,16 +1,26 @@
 package mate.academy.hibernate.relations.service.impl;
 
+import mate.academy.hibernate.relations.dao.ActorDao;
+import mate.academy.hibernate.relations.exception.DataProcessingException;
 import mate.academy.hibernate.relations.model.Actor;
 import mate.academy.hibernate.relations.service.ActorService;
 
 public class ActorServiceImpl implements ActorService {
+    public static final String NO_ACTOR_WITH_SUCH_ID = "No actor with such id -> %d";
+    private final ActorDao actorDao;
+
+    public ActorServiceImpl(ActorDao actorDao) {
+        this.actorDao = actorDao;
+    }
+
     @Override
     public Actor add(Actor actor) {
-        return null;
+        return actorDao.add(actor);
     }
 
     @Override
     public Actor get(Long id) {
-        return null;
+        return actorDao.get(id).orElseThrow(
+                () -> new DataProcessingException(NO_ACTOR_WITH_SUCH_ID.formatted(id)));
```

Comment:
on Service layer use `EntityNotFoundException`
```suggestion
                () -> new EntityNotFoundException(NO_ACTOR_WITH_SUCH_ID.formatted(id)));
```